cscs-checks/apps/namd/namd_check.py

```diff
@@ -31,9 +31,16 @@ def __init__(self, variant='cpu', **kwargs):
             self.num_tasks = 16
             self.num_tasks_per_node = 1
 
-        self.sanity_patterns = sn.assert_eq(sn.count(sn.extractall(
-            r'TIMING: (?P<step_num>\S+)  CPU:', self.stdout, 'step_num')), 50)
-
+        energy = sn.avg(sn.extractall(r'ENERGY:(\s+\S+){10}\s+(?P<energy>\S+)',
+                        self.stdout, 'energy', float))
+        energy_reference = -2451359.5
+        energy_diff = sn.abs(energy-energy_reference)
```

Comment:
Please leave whitespace around the `-` operator. Check out the [coding style](https://github.com/eth-cscs/reframe/wiki/Coding-Style-Guide#formatting-operators).